src/transformers/models/resnet/test.py

```diff
@@ -0,0 +1,10 @@
+from transformers import ResNetBackbone, ResNetConfig
+import torch
+
+model = ResNetBackbone(ResNetConfig(out_features=["stage2", "stage3"]))
+
+pixel_values = torch.randn((1, 3, 224, 224))
+
+outputs = model(pixel_values)
+
+print(outputs.keys())
```

Comment:
@sgugger I'm almost done fixing all tests for the new backbone API, but it's a mystery to me why the `keys()` shown here on the last line is empty. Don't classes that inherit from `ModelOutput` contain a keys() - which in this case should include 'feature_maps'? 

Weirdly, typing `outputs.feature_maps` works just fine, and includes a tuple of FloatTensors.

Reply:
Should be fixed by #20416